SocialCareCaseViewerApi.Tests/V1/IntegrationTests/UpdateWorkerDetails.cs

```diff
@@ -75,25 +76,30 @@ public async Task UpdateWorkerWithNewTeamReturnsTheOnlyTheUpdatedTeam()
         [Test]
         public async Task UpdateWorkerWithNewTeamUpdatesAnyAllocationsAssociated()
```

Comment:
This has been changed in the new worker update workflow/branch. Existing allocations should not be touched in any way when worker moves team. Just something to bear in mind when merging